### PR TITLE
add redirects for old dataset pages

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: _layouts/redirect.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      <meta charset="utf-8"/>
+      <meta http-equiv="refresh" content="1;url={{ page.redirect }}"/>
+      <link rel="canonical" href="{{ page.redirect }}"/>
+      <script type="text/javascript">
+              window.location.href = "{{ page.redirect }}"
+      </script>
+      <title>Page Redirection</title>
+  </head>
+  <body>
+  If you are not redirected automatically, follow <a href='{{ page.redirect }}'>this link</a>.
```
Comment (beets) [inline _layouts/redirect.html:12]: Maybe add a note here: 

"The content has moved. If you are not redirected automatically, ..."

**Review by beets — APPROVED** [review]
awesome -- thanks for adding this!!

File: _layouts/redirect.html (hunk 1 above)
Comment (n-h-diaz) [reply _layouts/redirect.html]: done

**Comment by n-h-diaz** [comment]
thanks!

**Pull request merged**